app/views/shared/_country_picker.html.erb

```diff
@@ -2,6 +2,7 @@
   <fieldset class="govuk-fieldset country-picker js-country-picker govuk-!-font-size-16">
     <div class="country-picker-container">
       <%= f.hidden_field :render_errors, value: false %>
+      <%= f.hidden_field :anchor, value: nil %>
```

Comment:
You shouldn't need this? Unless @jebw knows something I don't?

Reply:
This is being set here: $(this.selectElement).siblings('input[name$="[anchor]"]').val(window.location.hash.substring(1)); in app/webpacker/src/javascripts/commodities.js

Reply:
So we're setting a hidden field with some javascript. I guess my point is I can't conceive of why we need to when either the referrer or the request url in the rails controller action is aware of this? Isn't it a) duplicating what we've already got in the wrong place and b) therefore adds unnecessary extras.

A page anchor also has 0 to do with the trading partner so it smells a bunch

Why can't you just use the goods_nomenclature_path method to do this for you? Is there a bug in it?

Reply:
I get this now:

1. The click event to change tab, updates the anchor client side but isn't visible to the server
2. We need to make sure it is visible to the server so that we can redirect to the right tab on the commodities page
3. This does not touch the trading partners page (even though it could but we'd need the referrer anchor for this to work on the javascript side)